src/lm/kaldi-lmtable.cc

```diff
@@ -207,15 +208,24 @@ bool LmTable::ReadFstFromLmFile(std::istream &istrm,
     }
     // found valid line
     ngram_order = atoi(inpline.substr(pos1+5, pos2-(pos1+5)).c_str());
+    orders.push_back(ngram_order);
     if (ngram_order > max_ngram_order) {
       max_ngram_order = ngram_order;
     }
   }
+  std::sort(orders.begin(), orders.end());
```

Comment:
Sorry, I failed to realize this, I was halfway to sleep. As Dan said, this
is fine. y.

On Wed, Nov 25, 2015 at 11:12 PM, Minhua722 <notifications@github.com>
wrote:

> In src/lm/kaldi-lmtable.cc
> <https://github.com/kaldi-asr/kaldi/pull/365#discussion_r45943079>:
>
> >      if (ngram_order > max_ngram_order) {
> >        max_ngram_order = ngram_order;
> >      }
> >    }
> > +  std::sort(orders.begin(), orders.end());
>
> @jtrmal <https://github.com/jtrmal> Currently I just checked if
> statistics of n-grams is provided. Is it necessary to check real count and
> the number from the header?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/365/files#r45943079>.
>